doc-content/enterprise-only/optimizer/quarkus-creating-proj-proc.adoc

```diff
@@ -69,58 +71,6 @@ h| Description
 +
 The values of the `projectGroupID` and the `projectArtifactID` properties are used to generate the project version. The default project version is `1.0.0-SNAPSHOT`. The values of the `platformGroupId` and `platformVersion` properties are used by `quarkus-universe-bom` to manage project dependencies.
 
```

Comment:
Do you plan to remove the result of the command completely or we better consider the updated version of it?

Reply:
Should I remove quarkus-universe-bom? Replace with anything?

Reply:
So the maven command should be 
```
mvn io.quarkus.platform:quarkus-maven-plugin:{QUARKUS_BOM}:create \
    -DprojectGroupId=com.example \
    -DprojectArtifactId=optaplanner-quickstart \
    -Dextensions="resteasy,resteasy-jackson,optaplanner-quarkus,optaplanner-quarkus-jackson" \
    -DnoExamples
cd optaplanner-quickstart
```
Parameters to remove from the table below
 `platformGroupId`
| The Group ID of the target platform.
| `platformVersion`
| The version of the platform that you want the project to use.
| `noExamples`
|

```
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>io.quarkus.platform</groupId>
        <artifactId>quarkus-bom</artifactId>
        <version>{QUARKUS_BOM}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
      <dependency>
        <groupId>io.quarkus.platform</groupId>
        <artifactId>quarkus-optaplanner-bom</artifactId>
        <version>{QUARKUS_BOM}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
  <dependencies>
    <dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-resteasy</artifactId>
    </dependency>
    <dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-resteasy-jackson</artifactId>
    </dependency>
    <dependency>
      <groupId>org.optaplanner</groupId>
      <artifactId>optaplanner-quarkus</artifactId>
    </dependency>
    <dependency>
      <groupId>org.optaplanner</groupId>
      <artifactId>optaplanner-quarkus-jackson</artifactId>
    </dependency>

    <dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-junit5</artifactId>
      <scope>test</scope>
    </dependency>
  </dependencies>
```

Reply:
Look at the adjusted Quarkus post https://quarkus.io/guides/optaplanner

Reply:
I updated the section. Please have a look.

Reply:
quarkus-maven-plugin:1.11.6.Final-redhat-00001:create  in maven command would be substituted with Quarkus product?

Reply:
<version>${quarkus.platform.version}</version> should be static, right

Reply:
Right.